src/content/ui-api/gsctl/migrate.md

```diff
@@ -0,0 +1,57 @@
+---
+linkTitle: Migration to kubectl gs
+title: Migrating from gsctl to kubectl gs
```

Comment:
Should we have `kubectl-gs` or `kubectl gs` here?

Reply:
Done